docs/source/tutorials/odsc_tutorial/sagemaker_backend.rst

```diff
@@ -0,0 +1,100 @@
+Distributed Tuning
+==================
+
+The second approach to shorten the time to the next decision is to decrease
+the *time per experiment*. This can be done, to some extent, by increasing the
+number of workers, i.e. the number of trials which are run in parallel. In
+this section, we show how this can be done.
+
+Comparing Different Numbers of Workers
+--------------------------------------
+
+Our `study above <comparison.html#a-comparative-study>`__ was done with 4
+workers. With the local backend, an experiment with all its workers runs on a
+single instance. We need to select an instance type with at least 4 GPUs, and
+each training script can use one of them only.
+
+Syne Tune provides another backend,
+:class:`~syne_tune.backend.SageMakerBackend`, which executes each trial as a
+separate SageMaker training job. This allows you to decouple the number of
+workers from the instance type. In fact, for this backend, the default
+instance type for our benchmark is ``ml.g4dn.xlarge``, which has a single
+GPU and is cheaper to run than ``ml.g4dn.12xlarge`` we used with the
+local backend above.
+
+In order to showcase the SageMaker backend, we run a second study in order
+to compare our 4 methods ``RS``, ``BO``, ``ASHA``, and ``MOBSTER`` using
+a variable number of workers (2, 4, 8). Here, ``max_wallclock_time`` is 5
+hours for 4, 8 workers, but double that (10 hours) for 2 workers. Using the
```

Comment:
Why is the wallclock time constant for 4 and 8 workers, but only changes for the 2 worker case?

Reply:
The default is 4, and scaling is done only when `n_workers` is below that. It is a bit of a funny feature.